scripts/xapi-tracing-log-trim.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+export MAX_MB=100
+
+# find the oldest trace files beyond MAX_MB size and delete
+find /var/log/dt -type f -printf "%T@ %s %p\n" | sort -nr | awk 'BEGIN{size=0}{size+=$2; if (size>1024*1024*MAX_MB) {$1=""; $2=""; print $0}}' | xargs rm
+
+# delete any remaining files that are older than 30 days
+find /var/log/dt -mtime +30 -delete
```

Comment:
Would it make sense to delete more traces if the total amount of traces is bigger than a set amount?

Reply:
traces can also vary on size, the entire reason to delete them is to avoid too much space being used up. Using size concretely caps how much can be stored whereas with a set amount of traces that wouldn't be as clear.

Reply:
Sorry if it wasn't clear, I meant a set amount in terms of disk space, not number of files

Reply:
The set amount is 100mb